internal/handlers/common/aggregations/stages/statistics.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package aggregations
+package stages
```

Comment:
Why did we move `statistics.go` but didn't move `pushdown.go`?

Reply:
Moved both to `stages`.